paasta_tools/cli/cmds/mark_for_deployment.py

```diff
@@ -783,7 +999,8 @@ def wait_for_deployment(service, deploy_group, git_sha, soa_dir, timeout):
                  .format(git_sha, deploy_group))
 
     deadline = time.time() + timeout
-    green_light = Event()
+    if green_light is None:
+        green_light = Event()
```

Comment:
Is this safe to do even if there are not threads going on?

Reply:
Yep, pretty sure.